docs/api/add-on/dataModelTranslation-generic-http.md

```diff
@@ -0,0 +1,107 @@
+# dataModelTranslation IDD
+**generic_http & generic_https**
+
+## Overview
+
+This page describes the [generic_http](../communication-profiles/generic-http-template.md) and [generic_https](../communication-profiles/generic-https-template.md) service interface of dataModelTranslation, which enables translation between different data models.
+
+Hereby the **Interface Design Description** (IDD) is provided to the [dataModelTranslation – Service Description](../../assets/sd/5_1_0/dataModelTranslation_sd.pdf). For further details about how this service is meant to be used, please consult that document.
+
+## Interface Description
+
+### init-translation
+
+The service operation **request** may require an [access token](../primitives.md#accesstoken) (depending on the applied [service security](../../help/service-security.md)) and a [DataModelTranslationInitRequest](../data-models/data-model-translation-init-request.md) JSON encoded body.
```

Comment:
not true. We don't support tokens in case of data model translators. The TranslationManager specifically searches for data model translators with security policy NONE (or CERT if the TranslationManager uses certificate itself).